dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -4951,6 +4900,26 @@ public void validateContentlet(Contentlet contentlet,
 
     }
 
+    private void validateIdentifier(final Contentlet contentlet) {
+
+        if (UtilMethods.isSet(contentlet.getIdentifier())) {
+
+            boolean isIdentifier = false;
+
+            try {
+                isIdentifier = APILocator.getIdentifierAPI().isIdentifier(contentlet.getIdentifier());
```

Comment:
Is performance something we care about here? If so we could use the shortyapi. It does this lookup and also caches the result.
Usage would be `shortyAPI.getShorty(contentlet.getIdentifier()).isPreset() && shortyAPI.getShorty(contentlet.getIdentifier()).get().shortType==ShortType.Identifier)`

Reply:
done thanks